src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/TupleTests.cs

```diff
@@ -331,5 +402,73 @@ .maxstack  1
                 locals.Free();
             });
         }
+
+        [WorkItem(13803, "https://github.com/dotnet/roslyn/issues/13803")]
+        [Fact]
+        public void AliasElement_NoNames()
+        {
+            var source =
+@"class C
+{
+    static (int, int) F;
+    static void M()
+    {
+    }
+}";
+            var comp = CreateCompilationWithMscorlib(source, new[] { ValueTupleRef }, options: TestOptions.DebugDll);
+            WithRuntimeInstance(comp, new[] { MscorlibRef, SystemCoreRef, SystemRuntimeFacadeRef, ValueTupleRef }, runtime =>
+            {
+                var context = CreateMethodContext(runtime, "C.M");
+                var alias = new Alias(
+                    DkmClrAliasKind.Variable,
+                    "x",
+                    "x",
+                    "System.ValueTuple`8[" +
+                        "[System.Int32, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]," +
```

Comment:
Any way to avoid version numbers and tokens in this test? What happens if we update the checked in mscorlib?